test/SwiftSyntax/SyntaxCollections.swift

```diff
@@ -0,0 +1,98 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+// REQUIRES: OS=macosx
+// REQUIRES: objc_interop
+
+import Foundation
+import StdlibUnittest
+import SwiftSyntax
+
+var SyntaxCollectionsAPI = TestSuite("SyntaxCollectionsAPI")
+
+SyntaxCollectionsAPI.test("AppendingElement") {
+    let arrayElementList = SyntaxFactory.makeArrayElementList([
+        SyntaxFactory.makeArrayElement(expression: SyntaxFactory.makeIntegerLiteral("0"), trailingComma: nil)
```

Comment:
`makeIntegerLiteral` returns a `TokenSyntax`, not an `ExprSyntax`. Can you rewrite this to use `makeIntegerLiteralExpr`?

You might want to make a helper:

```swift
func integerLiteralElement(_ int: Int) -> ArrayElementSyntax {
  let literal = SyntaxFactory.makeIntegerLiteral("\(int)")
  return SyntaxFactory.makeArrayElement(
    expression: SyntaxFactory.makeIntegerLiteralExpr(digits: literal),
    trailingComma: nil)
}
```

That way you always know you're constructing a full `ArrayElementSyntax`.